src/GoTrueClient.ts

```diff
@@ -2085,12 +2087,19 @@ export default class GoTrueClient {
       // finished and tests run endlessly. This can be prevented by calling
       // `unref()` on the returned object.
       ticker.unref()
+      willUnref = true
       // @ts-ignore
     } else if (typeof Deno !== 'undefined' && typeof Deno.unrefTimer === 'function') {
       // similar like for NodeJS, but with the Deno API
       // https://deno.land/api@latest?unstable&s=Deno.unrefTimer
       // @ts-ignore
       Deno.unrefTimer(ticker)
+      willUnref = true
+    }
+
+    if(!willUnref && !GoTrueClient.memoryLeakWarningShown) { // prevents the warning from being spammed / shown multiple times
```

Comment:
Please use a single global constant about this, not per-client. Some environments, like Next.js for example, create a client on every request and this would pollute logs.

Reply:
Also please don't show it when `isBrowser()` returns true.

Reply:
@hf I believe this is only shown once, because it uses the static/class variable `memoryLeakWarningShown` (**not** a client/instance variable). This is modeled based on the approach used for https://github.com/supabase/auth-js/blob/99821f4a1f6fdb3a222cd0f660210016e6cc823e/src/GoTrueClient.ts#L117

I'll make the `isBrowser` change now.